python/ray/rllib/agents/ddpg/ddpg_policy_graph.py

```diff
@@ -189,13 +239,16 @@ def __init__(self, observation_space, action_space, config):
 
         # Action outputs
         with tf.variable_scope(A_SCOPE, reuse=True):
-            deterministic_flag = tf.constant(value=False, dtype=tf.bool)
+            deterministic_flag = tf.constant(value=True, dtype=tf.bool)
```

Comment:
I guess the value of this doesn't matter since eps is zero?

Reply:
yes, but the following output_actions are used for target Q value and BP grads to policy net so "deterministic" is more natural. eps applies to OU now. I implemented the Gaussian noise according to the original paper and baselines where the scale of action space and eps has not been considered.

Reply:
Ah I see, that sounds fine to not include it.